core/__tests__/tasks/profileProperty/enqueue.ts

```diff
@@ -84,9 +84,7 @@ describe("tasks/profileProperties:enqueue", () => {
           const pendingProfileProperties = await ProfileProperty.findAll({
             where: { state: "pending" },
           });
-          expect(pendingProfileProperties.length).toBe(
-            (propertiesCount - 1) * 2
-          );
+          expect(pendingProfileProperties.length).toBe(propertiesCount * 2);
```

Comment:
It's a side-effect, but this PR also makes this much easier to reason about!  If there are 10 Properties and 2 Profiles, there should be 20 pending Profile Properties.